internal/cluster/transition_test.go

```diff
@@ -175,7 +173,7 @@ var _ = Describe("Transition tests", func() {
 			{
 				name:                       "available builtin operators, progress OLM -> finalizing",
 				uploadKubeConfig:           true,
-				updateSuccessfullyFinished: true,
+				updateSuccessfullyFinished: false,
```

Comment:
This feels like it's significantly changing the expectations of this test.
Should we look at if this is a bug?

Reply:
No objection to handle this as a bug if needed. But I believe that the test is incorrect: a cluster that is in "finalizing" isn't completely installed. Is that wrong?

Reply:
as far as i know OLM is not a blocker for installation finalizing mean is waiting for required operators. 
@gamli75 maybe you are familiar with that more them me

Reply:
@filanov note that this test is checking a transition from "finalizing" to "finalizing", and I think that is a nop transition, so it doesn't update metrics.

Reply:
as @filanov wrote - OLM is not a blocker for successful installation

Reply:
@gamli75 you are right, I misunderstood the reason why this test is not updating the metrics. The reason is that the test transitions from the "finalizing" state to the same "finalizing" state, and that is a no-op transition, so nothing is done, in particular no metric is updated. So I still think this change is correct.